project/UTS46IDNAMappingTable.scala

```diff
@@ -0,0 +1,920 @@
+package org.typelevel.idna4s.build
+
+import cats._
+import cats.data._
+import cats.syntax.all._
+import java.io.File
+import java.io.FileOutputStream
+import java.net.URL
+import java.nio.charset.StandardCharsets
+import sbt.Keys.baseDirectory
+import sbt._
+import scala.annotation.tailrec
+import scala.io.Source
+import scala.meta._
+import scala.util.Failure
+import scala.util.Success
+import scala.util.Try
+import scala.util.matching._
+import scala.collection.immutable.SortedSet
+import scala.collection.immutable.SortedMap
+
+/** A plugin for generating the UTS-46 lookup tables for step 1 of UTS-46
+  * processing.
+  *
+  * The plugin does not automatically run. It should only need to be run when
```

Comment:
I'll need to update this if we move this to `src_managed`, as it will need to be always run.